addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2986,6 +2986,134 @@ var FieldProgressBar = AbstractField.extend({
+ * - title: title of the bar, displayed on top of the bar options
+ * - step: Round the clicked value on the progressbar to the nearest multiple of step (default: 5, min: 1)
+ */
+var FieldBasicProgressBar = AbstractField.extend({
+    description: _lt("Basic Progress Bar"),
+    template: "BasicProgressBar",
+    events: {
+        'click .o_progress': '_onClick',
+        'keydown .o_progress_keydown': '_onKeydown',
+        'mousedown .o_progress': '_onMousedown',
+        'mousemove .o_progress': '_onMousemove',
+        'mouseout .o_progress': '_onMouseout',
+    },
+    supportedFieldTypes: ['integer', 'float'],
+    init: function () {
+        this._super.apply(this, arguments);
+        this.canWrite = !this.nodeOptions.readonly && this.mode === 'edit';
+        this.max_value = this.recordData[this.nodeOptions.max_field] || this.nodeOptions.max_num || 100;
+        this.step = this.nodeOptions.step !== undefined ? this.nodeOptions.step : 5;
+        this.title = _t(this.attrs.title || this.nodeOptions.title) || '';
+        this.moused_down = false;
+    },
+    _render: function () {
+        this._render_value();
+        return this._super();
+    },
+    /**
+     * Updates the widget with value
+     *
+     * @param {Number} value
+     */
+    on_update: function (value) {
+        // _setValues accepts string and will parse it
+        var formattedValue = this._formatValue(value);
+        this._setValue(formattedValue);
+    },
+    /**
+     * Get the value of the click on the progress bar
+     * 
+     * @param {Event} event 
+     */
+    get_progress_value: function (event) {
+        var $target = $(event.currentTarget);
+        var numValue = Math.floor((event.pageX - $target.offset().left) / $target.outerWidth() * this.max_value);
+        if(this.step > 0){
+            numValue = Math.round(numValue/this.step)*this.step;
+        }
+        return numValue;
+    },
+    /**
+     * Renders the value
+     *
+     * @private
+     * @param {Number} v
+     */
+    _render_value: function (v) {
+        var value = this.value;
+        var max_value = this.max_value;
+        if (!isNaN(v)) {
+            value = v;
+        }
+        value = value || 0;
+        max_value = max_value || 0;
+
+        var widthComplete;
+        if (value <= max_value) {
+            widthComplete = value/max_value * 100;
+        } else {
+            widthComplete = 100;
+        }
+
+        this.$('.o_progress').toggleClass('o_progress_overflow', value > max_value)
+            .attr('aria-valuemin', '0')
+            .attr('aria-valuemax', max_value)
+            .attr('aria-valuenow', value);
+        this.$('.o_progressbar_complete').css('width', widthComplete + '%');
+
+        if (max_value !== 100) {
+            this.$('.o_progressbar_value').text(utils.human_number(value) + " / " + utils.human_number(max_value));
+        } else {
+            this.$('.o_progressbar_value').text(utils.human_number(value) + "%");
```

Comment:
in the option it could be nice to set the unit (for exemple : '%').
if max is 100 it is not always a percent.

Reply:
Yes, it's a good idea ! Thanks !

I fixed it in : https://github.com/odoo/odoo/pull/64239/commits/8c5abc1e7f89a5201b0b33017a1ba365a8fafd0e